docs/getting-started/symfony-bundle.md

```diff
@@ -295,7 +294,7 @@ class HomePage extends AbstractController
 {% block title %}Site A - Homepage{% endblock %}
 
 {% block content %}
-    {% if is_feature_enabled('feature') %}
+    {% if 'feature' is enabled %}
```

Comment:
The second and third parameters of the `is_feature_enabled` function are optional. When not passed the function is equivalent to the `enabled` test.

Maybe we can show it explicitly rewriting this as 
```suggestion
    {% if 'feature' is enabled and is_feature_enabled('feature') %}
```
or
```suggestion
    {% if 'feature' is enabled %}{# could be written as "if is_feature_enabled('feature')" #}
```

Reply:
I will not mix both examples, we can add new examples explaining each case. Or also, add a page explaining only the twig extension. Think that it is a very good candidate to be a standalone package because Twig can be used in any PHP application.